config/services.xml

```diff
@@ -143,5 +144,11 @@
             <tag name="messenger.message_handler"/>
         </service>
 
+        <service id="webgriffe_sylius_akeneo.product.validator" class="Webgriffe\SyliusAkeneoPlugin\Product\Validator">
```

Comment:
Maybe a more right name could be `webgriffe_sylius_akeneo.validator.product`? And as a consequence also the namespace should become `Webgriffe\SyliusAkeneoPlugin\Validator\Product`?

Reply:
Mmmm no... All Product importer stuff are in the `Webgriffe\SyliusAkeneoPlugin\Product` namespace.